test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -27,7 +27,6 @@
 import com.carrotsearch.randomizedtesting.generators.RandomNumbers;
 import com.carrotsearch.randomizedtesting.generators.RandomPicks;
 import com.carrotsearch.randomizedtesting.generators.RandomStrings;
-
```

Comment:
Could you revert this line removal? I don't mind changing how imports are shaped but I don't think we should do it in files that we're not otherwise editing.